modules/fscloud/README.md

```diff
@@ -1,16 +1,18 @@
 # Pre-wired CBR configuration for FS Cloud
 
 This module creates default coarse-grained CBR rules in a given account following a "secure by default" approach - that is: deny all flows by default, except known documented communication in the [Financial Services Cloud Reference Architecture](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-vpc-architecture-about):
-- COS -> KMS
-- Block storage -> KMS
-- ROKS -> KMS
+- COS -> HPCS
+- Block storage -> HPCS
+- ROKS -> HPCS
 - Activity Tracker route -> COS
 - VPCs where clusters are deployed -> COS
 - IS (VPC Infrastructure Services) -> COS
 - VPCs -> container registry
-- All ICD -> KMS
+- All ICD -> HPCS
 - IKS -> IS (VPC Infrastructure Services)
 
+This module now supports Hyper Protect Crypto Services. By default it is set to use Hyper Protect Crypto Services but this can be modified to use Key Protect or both Key Protect and Hyper Protect Crypto Services using the terraform variable `key_protect`.
```

Comment:
the name of the variable is 'kms' though? key-protect is the value.

Reply:
Done.